internal/evaluator/__testdir__/a.rego

```diff
@@ -0,0 +1,28 @@
+# A set of policies
+package a
+
+# METADATA
+# custom:
+#   short_name: failure
+deny[result] {
+	result := {
+		"code": "main.failure",
```

Comment:
Should the code use the package name instead of `main` to better simulate our policies?